src/app/app.module.ts

```diff
@@ -15,6 +16,7 @@ import { CoreModule } from './core/core.module';
     AppRoutingModule,
     HomeModule,
     LayoutModule,
+    WhoDidThisModule
```

Comment:
@lizzymendivil  No need to add the module in the main module as you are using lazy loading. Lazy loading feature loads this module when the route is actually open, if we include this in the main app.module basically we are loading it from the beginning without a reason. 

I will merge this but please send the fix for this in another pr.